security-jwt/src/main/java/io/micronaut/security/token/jwt/signature/jwks/JwksSignature.java

```diff
@@ -182,30 +226,60 @@ protected JWKSet loadJwkSet(String url) {
      * @param jwt A Signed JWT
      * @param jwkSet A JSON Web Key Set
      * @param refreshKeysAttempts Number of times to attempt refreshing the JWK Set
+     * @deprecated Use {@link JwksSignature#matches(SignedJWT, JWKSet)} instead
      * @return a List of JSON Web Keys
      */
+    @Deprecated
     protected List<JWK> matches(SignedJWT jwt, @Nullable JWKSet jwkSet, int refreshKeysAttempts) {
+        return matches(jwt, jwkSet);
+    }
+
+    /**
+     * Calculates a list of JWK matches for a JWT.
+     *
+     * Since the JWTSet is cached it attempts to refresh it (by calling its self recursive)
```

Comment:
```suggestion
     * Since the JWKSet is cached it attempts to refresh it (by calling itself recursively)
```

Reply:
Also it doesn't seem to call itself recursively anymore, is this just an out of date comment?

Reply:
I have remove the line from the java doc. Please, note that I have deprecated the method.

Reply:
thanks, I removed it.